app/Http/Controllers/Bible/BiblesController.php

```diff
@@ -186,6 +201,9 @@ public function index()
                     $q->where('organization_id', $organization->id);
                 });
             })
+            ->when($audio_timing, function ($q) {
```

Comment:
is there a reason you couldn't add a filter, similar to filterByLanguage? For each bible, we just need to join to bible_file_timestamps and use that to eliminate bibles which don't have timestamps.

Reply:
It is related to the case when the filter is empty so, it should NOT add the query part to add the bible_file_timestamps table.